File: scala/BeginnersSelection/01_Product.scala
// https://atcoder.jp/contests/abs/tasks/abc086_a

object Main {

  implicit class Pipeline[T](x: T) {
    def |>[S](f: T => S): S = f(x)
  }

  def main(args: Array[String]) = (
    io.StdIn.readLine
      .split(" ")
      .map(_.toInt)
      .map(_ & 1)
      .reduce((a, x) => a & x)
      |> ((x) => if (x == 1) "Odd" else "Even")
      |> println
  )
}

// https://atcoder.jp/contests/abs/submissions/19753942

File: kt/BeginnersSelection/01_Product.kt
// https://atcoder.jp/contests/abs/tasks/abc086_a

fun main(args: Array<String>) = (
  readLine()!!
    .split(" ")
    .map(String::toInt)
    .map { it and 1 }
    .reduce { a, x -> a and x }
    .let { if (it == 1) "Odd" else "Even" }
    .let(::println)
)

// https://atcoder.jp/contests/abs/submissions/21020996
